platform/web/os_web.cpp

```diff
@@ -152,6 +152,13 @@ bool OS_Web::_check_internal_feature_support(const String &p_feature) {
 	if (p_feature == "web") {
 		return true;
 	}
+	if (p_feature == "simd") {
+#ifdef SIMD_ENABLED
+		return true;
+#else
+		return false;
+#endif
+	}
```

Comment:
Is this feature tag needed anywhere? In other words, is there a scenario where users could benefit from using `OS.has_feature("simd")`?

Reply:
That's a good question. I used it for the tests, it was pretty useful.

Technically, if you create a GDExtension, it should be built with the same SIMD settings, as it is another type of build.

Reply:
I can remove the tag! Or renaming it `wasm_simd`.